src/classes/bootstrap.php

```diff
@@ -591,6 +595,11 @@ public function after_setup_theme_early_bind() {
 			class_alias( 'PMC\Unit_Test\Traits\Singleton', 'PMC\Global_Functions\Traits\Singleton' );
 		}
 
+		// Point deprecated \Requests class (pre-6.2) over to new one.
+		if ( ! class_exists( 'WpOrg\Requests\Requests', false ) ) {
+			class_alias( 'Requests', 'WpOrg\Requests\Requests' );
```

Comment:
This adds backwards compatibility for the deprecated `\Requests` while it's still in WordPress Core.